src/zope/testing/setupstack.txt

```diff
@@ -239,6 +238,8 @@ Here's an example:
     ...             print('done w test')
     ...
     ...     def test(self):
+    ...         if os.getcwd() == curdir:
+    ...             print('Failed to change directory')
     ...         print(os.listdir('.'))
```

Comment:
Maybe drop the `print(os.listdir('.'))` and its output below?

Reply:
I suppose so.